Gemfile

```diff
@@ -62,6 +62,8 @@ if RUBY_VERSION >= '2.6.0'
   gem 'rubocop-rspec', ['~> 2.20', '< 2.21'], require: false
 end
 
+gem 'ostruct' if RUBY_VERSION >= '3.3.5'
```

Comment:
Minor: I suggest maybe doing `>= 3.3` instead of `3.3.5`.

My thinking is that we're not very consistent in upgrading to point releases in our different test/development environments, and on the off chance we have some weird issue from this, it may be a bit harder to diagnose if only some of the 3.3 series gets this new version.

Reply:
fixed here [261b590](https://github.com/DataDog/dd-trace-rb/pull/3890/commits/261b59017ec0a670e3aea1e99a901b4ced0bdda0) (force pushed)